### PR TITLE
Fix: escape colname if using reserved keywords

### LINKED ISSUE #8: If using a reserved keywords as a column
Nice to meet you. I have a question this time and I have created an issue.
First of all, thanks for creating such a wonderful tool.
It is very helpful.

I have a question: 
If I use reserved words as columns, may need to backquote them when I issue the query.
[https://cloud.google.com/spanner/docs/data-definition-language#reserved_keywords](https://cloud.google.com/spanner/docs/data-definition-language#reserved_keywords)

In automatically generated code, reserved words are not enclosed in backquotes, which results in an error when the function is executed.
```
spanner: code = "InvalidArgument", desc = "Syntax error: Unexpected keyword HASH...
```

Is there any way to avoid this error?
If not, do you plan to implement it in the future?

If there is no plan to implement it, is it possible to merge the PR after I fix it?

I look forward to your reply.
Kind Regards,

### PR DESCRIPTION
**Pull request by nandehu0323**
close #8

Please read the CLA carefully before submitting your contribution to Mercari.
Under any circumstances, by submitting your contribution, you are deemed to accept and agree to be bound by the terms and conditions of the CLA.

https://www.mercari.com/cla/

### CODE REVIEW

File: internal/reserved_keywords.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
+package internal
+
+var reservedKeywords = map[string]struct{}{
```
Comment (kazegusuri) [inline internal/reserved_keywords.go]: Where can we have this list?

Comment (kazegusuri) [reply internal/reserved_keywords.go]: Please add a link for the list.

File: generator/funcs.go [hunk 2]
```diff
@@ -189,7 +190,31 @@ func (a *Generator) colnames(fields []*internal.Field, ignoreNames ...interface{
 		str = str + a.colname(f.Col)
 		i++
 	}
+	return str
+}
+
+// colnames creates a list of the column names found in fields, excluding any
```
Comment (kazegusuri) [inline generator/funcs.go]: Please fix the comments.

File: internal/reserved_keywords.go (hunk 1 above)
Comment (nandehu0323) [reply internal/reserved_keywords.go]: Thank you for your comment.
I fixed.
Please check it.

File: generator/funcs.go (hunk 2 above)
Comment (nandehu0323) [reply generator/funcs.go]: Thank you for your comment.
I fixed.
Please check it.

**Comment by kazegusuri** [comment]
Please squash the commits, then I will merge.

**Comment by kazegusuri** [comment]
thanks!!

**Pull request merged**